src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,162 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String PASSWORD_CORRECT = "Abcdfg";
+    private static final String PASSWORD_INCORRECT = "dfg";
+    private static final int AGE_CORRECT = 19;
+    private static final int AGE_BOUND = 18;
+    private static final int AGE_INCORRECT = 17;
+    private static final int AGE_NEGATIVE = -18;
+    private static RegistrationService service;
+    private static StorageDao storageDao;
+    private User actual;
+
+    @BeforeAll
+    static void beforeAll() {
+        service = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        actual = new User();
+    }
+
+    @Test
+    void register_nullUser_throwsException() {
```

Comment:
```suggestion
    void register_nullUser_NotOk() {
```
Make naming like this everywhere

Reply:
Done